libraries/adaptive-expressions/src/parser/expressionParser.ts

```diff
@@ -77,7 +77,9 @@ export class ExpressionParser implements ExpressionParserInterface {
                 result = new Constant(false);
             } else if (symbol === 'true') {
                 result = new Constant(true);
-            } else if (symbol === 'null' || symbol === 'undefined') {
+            } else if (symbol === 'null') {
+                result = new Constant(null);
+            } else if (symbol === 'undefined') {
```

Comment:
Are you sure we should make this change?  This will break existing functions that check for null--now it will have to be changed to check for both null and undefined and I don't think that is a distinction we want to make.